collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -0,0 +1,70 @@
+
+### Convert REPv1 to REPv2
+REPv1 holders **MUST** convert their [REPv1]({{glossary_repv1}}) to [REPv2]({{glossary_repv2}}). Because REP in the [parent universe]({{glossary_parent_universe}}) can be migrated to only its [child universe]({{glossary_child_universe}}).  (The name of REP version 1 is `REP` correctly, but to distinguish from version 2, we call it `REPv1` here.)
+
+That means they have to follow the chain of [universes]({{glossary_universe}}). For example, [forks]({{glossary_fork}}) occur in the following order:
+```
+universe A → universe B → universe C
+```
+and REP exits in the *universe A*, it has to be migrated as follows: 
+```
+universe A → universe B → universe C
+```
+It cannot skip `universe B`.
+{% capture image_src %}{{ "/" | absolute_url }}assets/images/{{page.collection}}/fork/to-dos/chain-of-universes.svg{% endcapture %}
+{% include zoom-image.html src=image_src caption="Figure 1. chain of universes" %}
+
+### Check monthly to see if a fork occurs
+[Forks]({{glossary_fork}}) last for 60 days, so it is strongly recommended that all REP holders "check in" on the project at least once a month to make sure that no fork is underway or they risk losing all of their REP.
+
+The process of "checking in" just means open the [Augur UI]({{ "/" | absolute_url }}/{{page.collection}}/8-augur-ui.html) or ask around (Discord, Twitter, Reddit, etc.) and see if a fork is underway. If no fork is underway then you can go back to passively holding for a month until your next check in without risk of losing your REP.
+
+Useful links for checking if a fork occurs are:
+ - [Augur Discord server](https://discord.gg/guXC9Bd4)  ⭐*recommended*⭐
+ - The [official Twitter account](https://twitter.com/augurproject)
+ - The [official blog](https://augur.net/blog)
+ - The subreddit [/r/Augur](https://www.reddit.com/r/Augur/)
+
+## After starting a fork (60 days)
+REP holders needs to pick a side and migrate their REP to the [child universe]({{glossary_child_universe}}) they believe aligns with reality within 60 days from starting a [fork]({{glossary_fork}}).
+
+After 60 days from starting a fork, [REP]({{glossary_reputation_token}}) in the [parent universe]({{glossary_parent_universe}}) are [locked]({{glossary_locked_universe}}) and can not be migrated to the child universe permanently.
+
+However, the choice should be considered carefully, because migration is one-way; it cannot be reversed. REP cannot be sent from one [sibling universe]({{glossary_sibling_universe}}])to another.
+
+### What if REP holders don't wish to join in the fork?
+They may sell their [REP]({{glossary_reputation_token}}) before the [fork]({{glossary_fork}}) is over, but beware that if many people exercise this strategy the price of REP may drop leading up to the fork and then rebound in the [winning universe]({{glossary_winning_universe}}) after the fork, this could be a costly strategy.
```

Comment:
It may rebound in the **true** universe, which may or may not be the winning universe.

Reply:
I submitted an issue https://github.com/msagansk/Augur.Guide/issues/39 .

Reply:
I will replace `winning universe` with `truth universe` instead of `true universe`, because `truth universe` is used in the [basic-knowledge](https://crystalballbe.github.io/version2-english/9-fork/1-basic-knowledge.html#what-happens-to-the-child-universes).

Reply:
I fixed it by https://github.com/msagansk/Augur.Guide/pull/40.